user/tests/wiring/cell_connect_after_off/application.cpp

```diff
@@ -0,0 +1,14 @@
+#include "application.h"
```

Comment:
Please add a README with prerequisites for running this test. AFAIK it's required to detach the antenna.